src/core/_modules/event_hubs/data.tf

```diff
@@ -0,0 +1,11 @@
+# TODO: remove when monitor module is implemented
+data "azurerm_monitor_action_group" "error_action_group" {
```

Comment:
i think we can move this data on the main module as generally few action group are used and then shared??